lib/gollum-lib/page.rb

```diff
@@ -281,6 +281,14 @@ def versions(options = {})
       @wiki.repo.git.versions_for_path(@path, @wiki.ref, log_pagination_options(options))
     end
 
+    # Public: The last version that has touched the Page. Can be nil.
+    #
+    # Returns Gollum::Git::Commit, or nil.
+    def last_version
+      return @last_version if defined? @last_version
```

Comment:
`#last_version` will only be `nil` if the page does not exist on disk yet. While the utility of returning a non-`nil` value in a preview is not very high, that is the current behavior so I kept it.